<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_page</class>
 <widget class="QWidget" name="widget_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vertThings"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
